x11-wm/cwm/cwm-6.6.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs
+
+DESCRIPTION="OpenBSD fork of calmwm, a clean and lightweight window manager"
+HOMEPAGE="https://www.openbsd.org/cgi-bin/cvsweb/xenocara/app/cwm/
+	https://github.com/chneukirchen/cwm"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/leahneukirchen/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/leahneukirchen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="ISC"
+SLOT="0"
+
+RDEPEND="x11-libs/libXft
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+"
+BDEPEND="${RDEPEND}
```

Comment:
no, this is incorrect.
you should define
DEPEND="${RDEPEND}",
BDEPEND should only contain bison and pkgconfig

Reply:
here's some documentation on differences between DEPEND and BDEPEND
https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html#build-time-dependencies-split-into-depend-and-bdepend

Reply:
Thanks a lot for the link!

I've split them correctly.